6-more-secure-builds/Dockerfile.multistage

```diff
@@ -0,0 +1,30 @@
+ARG NODE_VERSION=20
+
+FROM node:${NODE_VERSION}-alpine AS build
+
+ENV NODE_ENV=production
+ARG GITHUB_PAT
+ENV GITHUB_PAT=${GITHUB_PAT}
+
+WORKDIR /usr/src/app
+
+COPY .npmrc ./.npmrc
+COPY package.json package.json
+COPY package-lock.json package-lock.json
+
+
+RUN npm ci --omit=dev
+
+COPY **/node_modules node_modules
```

Comment:
Do you still need this line copy if the npm ci has run correctly within the build step on line 16?